CHANGELOG.md

```diff
@@ -69,7 +69,8 @@ Bottom level categories:
 - Flesh out docs. for `AdapterInfo::{device,vendor}` by @ErichDonGubler in [#3763](https://github.com/gfx-rs/wgpu/pull/3763).
 - Spell out which sizes are in bytes. By @jimblandy in [#3773](https://github.com/gfx-rs/wgpu/pull/3773).
 - On Web, types don't implement `Send` or `Sync` anymore. By @daxpedda in [#3691](https://github.com/gfx-rs/wgpu/pull/3691)
-- Validate that `descriptor.usage` is not empty in `create_buffer` by @nical in [https://github.com/gfx-rs/wgpu/pull/#3928](3928)
```

Comment:
nitpick: It'd be nicer for history if:

1. This fix of your previous `CHANGELOG` message were a separate commit.
2. The addition of the `CHANGELOG` entry for this PR were folded into the commit making the change.

Perhaps you were assuming that we'd squash? 🤔

Reply:
> fully independent commits

That's my goal with this particular feedback. 🙂